api/jwt/jwt.go

```diff
@@ -2,26 +2,31 @@ package jwt
 
 import (
 	"errors"
-
-	"github.com/portainer/portainer/api"
-
 	"fmt"
+	"github.com/portainer/portainer/api"
 	"time"
 
 	"github.com/dgrijalva/jwt-go"
 	"github.com/gorilla/securecookie"
 )
 
+const (
+	// change settings timeout accordingly, current 24h = 86400
```

Comment:
superfluous comment. Either leave it out or tell about the intention of this constant. Anyone could do the math, now that you provided the amount unit :)